uxarray/grid/grid.py

```diff
@@ -1912,3 +1919,27 @@ def isel(self, **dim_kwargs):
             raise ValueError(
                 "Indexing must be along a grid dimension: ('n_node', 'n_edge', 'n_face')"
             )
+
+    def get_dual(self):
+        """Compute the dual mesh for a grid, returns a new grid object.
+
+        Returns
+        --------
+        dual : Grid
+            Dual Mesh Grid constructed
+        """
```

Comment:
Can you add more information to the docstring?

Reply:
Sure! Thanks for the review, I will address this today and get those changes pushed!